exchanges/kraken/kraken_test.go

```diff
@@ -2196,7 +2196,7 @@ func TestGetFuturesContractDetails(t *testing.T) {
 	}
 
 	_, err = k.GetFuturesContractDetails(context.Background(), asset.Futures)
-	if !errors.Is(err, nil) {
+	if err.Error() != "USD index is not found" {
```

Comment:
Please revert as it causes panic [here](https://github.com/thrasher-corp/gocryptotrader/actions/runs/7220205770/job/19672667210?pr=1423#step:7:3372)

Reply:
Should be `assert.NoError()` which should not panic.
And generally we should rarely be outside of `assert.ErrorIs` and `assert.ErrorContains` where possible

Reply:
It was a typo, sorry about that. Added assert.NoError though. Let me know if you want to revert.